pkg/ccl/crosscluster/logical/logical_replication_job.go

```diff
@@ -531,6 +531,9 @@ func (rh *rowHandler) handleRow(ctx context.Context, row tree.Datums) error {
 			if md.RunStats != nil && md.RunStats.NumRuns > 1 {
 				ju.UpdateRunStats(1, md.RunStats.LastRun)
 			}
+			if l := rh.job.Details().(jobspb.LogicalReplicationDetails).MetricsLabel; l != "" {
+				rh.metrics.LabeledReplicatedTime.Update(map[string]string{"label": l}, replicatedTime.GoTime().Unix())
```

Comment:
hrm, maybe we should keep the option as label. i don't want the label key for our metric to be "metric_label".

Reply:
This string can be whatever we want; it doesn’t need to match the sql option. I don’t think it would be confusing to have the sql option “metric label” cause a “label” to appear _in metrics_. But that said, “metric” isn’t in the grammar and I don’t really want to add it.